src/main/resources/templates/c_plus_plus/test/testUtils/AbstractTest.py

```
@@ -0,0 +1,270 @@
+        # Register a function to raise a TimeoutError on the signal.
+        signal(SIGALRM, self.__raiseTimeout)
+        # Schedule the signal to be sent after ``time``.
+        alarm(timeoutSec)
+
+        with suppress(TimeoutError):
+            yield
+        # Unregister the signal so it won't be triggered
+        # if the timeout is not reached.
+        signal(SIGALRM, SIG_IGN)
+
+    def __raiseTimeout(self, sigNum: int, frame):
```

Ruff

Missing return type annotation for private function `__raiseTimeout`

Add return type annotation: `NoReturn`

(ANN202)